repos/system_upgrade/common/actors/scanpkgmanager/tests/test_scanpkgmanager.py

```diff
@@ -25,6 +24,12 @@ def mocked_get_releasever_path():
     return os.path.join(CUR_DIR, 'files/releasever')
 
 
+def mock_get_config_contents(config_contents):
+    def mocked_get_config_contents(path):
+        return config_contents
+    return mocked_get_config_contents
```

Comment:
This function can be defined locally, in the test that is using this function. Personally, I am against creating short functions that have less than `5` lines and are used only in one place.

The function could then be defined inside `test_get_proxy_if_set` as follows:
```python

@pytest.mark.parametrize('proxy_set', [True, False])
def test_get_proxy_if_set(monkeypatch, proxy_set):
    config_contents = 'Config go brrr'
    def mocked_get_config_contents(config_path):
        assert config_path == 'x'  # Check that the actor is getting contents of files you expect it to be
        return config_contents

    monkeypatch.setattr(scanpkgmanager, 'get_config_contents', mocked_get_config_contents) 
    

```